Sources/SwiftGenKit/Parsers/Strings/FileTypeParser/StringsFileTypeParser.swift

```diff
@@ -0,0 +1,19 @@
+//
+// SwiftGenKit
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import PathKit
+
+protocol StringsFileTypeParser: AnyObject {
+  init(options: ParserOptionValues)
+
+  var shouldOverwriteValuesInExistingTable: Bool { get }
```

Comment:
Just got an idea, but I think we can **keep that for a refining PR for the future** (so it's just to take note to improve the code later):

<details><summary>Idea: use concept of priorities instead of boolean</summary>

---

I think that instead of having this boolean variable in the protocol, we could consider that the parsers are listed in order of priority [in the  `StringsParser` property](https://github.com/SwiftGen/SwiftGen/pull/634/files#diff-3c7b36f20762bd985d9d78b5082c2493R61-R64) ; each entry would then be associated with a priority index when parsed, that priority index being the index of the parser in that `StringsParser` private array (so entries parsed with the `StringsFileParser` would have an associated priority index of 0 and files parsed with the `StringsDictFileParser` would have an associated priority index of 1); and then when we encounter an entry already existing for a given key we use the one with the higher priority index instead of relying on that current boolean property.

That approach would feel more logically correct to me (especially, imagine if we want to scale up that logic of multiple sub-parsers and one day we add a 3rd one, having a boolean is quite a binary decision, while the reality of the logic is more about the fact that one parser has more priority over the other), but I'm once again being pedantic and perfectionist here, I know 😇 .

This can totally wait for a separate PR as an idea to improve the code logic and do a small refactoring later. Just wanted to write it somewhere while I thought of the idea.

---

</details>